### DISCUSSION TITLE
Mention Miss_hit on the turing way book?

### DISCUSSION THREAD

**Discussion by Remi-Gau** [florianschanda/miss_hit#164] [General]
I was wondering if it would make sense to maybe mention all this work in the turing-way book.

They have a chapter on code quality where it would make a great addition.

https://the-turing-way.netlify.app/reproducible-research/code-quality.html

FYI: I can take care of it if you want. :-)

**Comment by florianschanda** [comment]
If you want to, sure.

Probably best you do it and not me, because there is a *lot* I would change/complain about there. They seem to think that static analysis == linter and verification == testing; and that's just omitting so much that it physically pains me ;)

**Comment by florianschanda** [comment]
To clarify: linting *is* static analysis, but there is more. You have 3 attributes:
* sound (we never miss a bug)
* complete (any bug we complain about really *is* a bug)
* automatic (no human cooperation required)

Linting tools are automatic, but neither sound nor complete.

They are ignoring overapproximate analysis (sound & automatic); underapproximate analysis (complete & automatic); and deductive verification (sound & complete).

And if they're aiming their things at the scientific community they probably should at least *mention* these.

**Comment by florianschanda** [comment]
@Remi-Gau were you doing to suggest an update to them? If so I'll leave it open and assign you. Otherwise I'll close the ticket (want to do some issue cleaning :))
